### DISCUSSION TITLE
unoapp does not create macOS AppKit project

### DISCUSSION THREAD

**Discussion by amsokol** [unoplatform/uno#12830] [Q&A]
Hello all,

Environment:
Apple M1 pro
Ventura 13.4.1
.NET 7.0.305
SDK Workloads are installed

`dotnet new unoapp -o MyApp01`
creates project solution with Mobile(iOS), Windows, Wasm, GTK projects but without macOS(AppKit) project.

`dotnet new unoapp -o MyApp01 -platforms macos`
does not work also.

Please help me to create MacOS(AppKit) project.

Thanks in advance.

**Comment by Martin1994** [comment]
Should be related to this PR https://github.com/unoplatform/uno/pull/11116
